src/rust/ide/view/graph-editor/src/component/visualization/foreign/java_script/instance.rs

```diff
@@ -231,6 +229,15 @@ impl Instance {
                     frp.data_receive_error.emit(Some(e));
                 }
              });
+
+            mouse_up       <- scene.mouse.frp.up.constant(());
+            mouse_down     <- scene.mouse.frp.down.constant(());
+            mouse_wheel    <- scene.mouse.frp.wheel.constant(());
+            mouse_position <- scene.mouse.frp.position.constant(());
+            caught_mouse   <- any(mouse_up,mouse_down,mouse_wheel,mouse_position);
+            should_process <- caught_mouse.gate(&frp.is_active);
+            trace should_process;
+            eval_ should_process ([scene] scene.frp.pass_processed_event_to_js.emit(()));
```

Comment:
So this is implemented in other way than we discussed (gather all info in graph_editor) and drive this setting from a single place. Why? What are the reasons behind this arch? Im not telling its bad, I was just sure we agreed tosoemthing else.

Reply:
So it seems that I recalled it wrongly as an "optional feature". I try to refactor that.

Reply:
After a call we decided to leave it as it is.